tljh-voila-gallery/tljh_voila_gallery/build_images.py

```diff
@@ -1,19 +1,19 @@
 
-import sys
+import logging
 import os
-from pathlib import Path
 import subprocess
+import sys
 from collections import namedtuple
-from pkg_resources import resource_stream
-import logging
 
+import docker
+import requests
 from ruamel.yaml import YAML
 
 logging.basicConfig(level=logging.INFO)
 
 yaml = YAML()
 
-GALLERY_PATH = 'gallery.yaml'  # Relative to module root
+GALLERY_URL = 'https://raw.githubusercontent.com/voila-gallery/gallery/master/tljh-voila-gallery/tljh_voila_gallery/gallery.yaml'
```

Comment:
How would you test this locally? What happens when the format of the yaml file changes but not the code? IMO we should (for now) still load it from the local package, and have some method of autodeployment instead.

Reply:
Good catch, we should of course support the local gallery.yaml. The same gallery file is also used on the front page so when a new example is added it should also be reflected there.

> What happens when the format of the yaml file changes but not the code? IMO we should (for now) still load it from the local package, and have some method of autodeployment instead.

Maybe one way to ensure consistency between the code and the gallery.yaml would be reinstall / upgrade the plugin? Followed by the build of the docker images.

Reply:
Rebased to revert back to reading from the local package.